File: Python & Ruby/Container/container.py
print(type('pxzhu'))
name = 'pxzhu'
print(name)
names = [
  'pxzhu',
  'Matt',
  'pea'
]
print(names)
print(names[2])
pxzhu = [
  'programmer',
  'daejeon',
  24,
  False
]
pxzhu[1] = 'busan'
print(pxzhu)

alpha = [
  'A',
  'B',
  'C'
]
print('A' in alpha) # True
print('D' in alpha) # False
beta = [
  1,
  100,
  12312312
]
print(len(beta)) # 3
print(min(beta)) # 1
print(max(beta)) # 12312312
del(beta[0])
print(beta) # [100, 12312312]
beta.append(1004)
print(beta)
# [100, 12312312, 1004]
beta.reverse()
print(beta)
# [1004, 12312312, 100]

File: Python & Ruby/Container/container.rb
names = [
  'pxzhu',
  'Matt',
  'pea'
]
puts(names.class)
puts(names[2])
names[0] = 'phs'
puts(names)
# phs
# Matt
# pea
print(names)
# ["phs", "Matt", "pea"]

alpha = [
  'A',
  'B',
  'C',
  'D'
]
puts(alpha.length) # 4
alpha.push('E')
print(alpha)
# ["A", "B", "C", "D", "E"]
alpha.delete_at(0)
print("\n")
print(alpha)
# ["B", "C", "D", "E"]